packages/AlgernonTests.package/ALGTestUI.class/instance/testAlgernonPositioningWindowIsSmallerAfterReopening.st

```diff
@@ -0,0 +1,9 @@
+testing
+testAlgernonPositioningWindowIsSmallerAfterReopening
+	| oldExtent |
+	oldExtent := World extent.
+	World extent: (100@100).
+	self algerInstance deactivate.
+	self algerInstance activate.
+	self assert: ((self algerInstance center x = self algerInstance class defaultPosition x rounded) & (self algerInstance center y = self algerInstance class defaultPosition y rounded)).
```

Comment:
You can just write `self algerInstance center =  self algerInstance class defaultPosition rounded`

Reply:
Did that.